package.json

```diff
@@ -50,6 +50,6 @@
   "scripts": {
     "prestart": "npm install",
     "start": "gulp --dev",
-    "build": "npm prestart && gulp"
+    "build": "npm run prestart && npm start"
```

Comment:
This works, but also causes `npm install` to run twice, since the `start` script automatically runs `prestart`. Can you change it to this instead?

```
"build": "npm run prestart && gulp"
```